common/thorhelper/thorsoapcall.cpp

```diff
@@ -1038,6 +1058,7 @@ class CWSCHelper : implements IWSCHelper, public CInterface
         if (wscType == STsoap)
```

Comment:
It would be worth adding an attribute for the activity id - that will be very useful to the developer.

activitySpanScope->setSpanAttrubute("activity.id", <n>)?

It might need to be a new parameter.

Reply:
This sounds useful, but couldn't find a reasonable way to pass that info down to the helper

Reply:
I added HPCC-31527 to add that as a separate task - it is relatively high priority.